modules/nw-modifying-operator-install-config.adoc

```diff
@@ -46,6 +46,19 @@ $ ./openshift-install create manifests --dir=<installation_directory> <1>
 <1> For `<installation_directory>`, specify the name of the directory that
 contains the `install-config.yaml` file for your cluster.
 
+. Modify the `manifests/cluster-scheduler-02-config.yml` Kubernetes manifest file to prevent Pods from being scheduled on the control plane machines:
```

Comment:
Just wondering if you should specify where this file is? Is it in the `<installation_directory>/manifests/` directory as mentioned in step 3?